homeassistant/components/airzone_cloud/entity.py

```diff
@@ -0,0 +1,54 @@
+"""Entity classes for the Airzone Cloud integration."""
+from __future__ import annotations
+
+from typing import Any
+
+from aioairzone_cloud.const import AZD_NAME, AZD_SYSTEM_ID, AZD_ZONES
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, MANUFACTURER
+from .coordinator import AirzoneUpdateCoordinator
+
+
+class AirzoneEntity(CoordinatorEntity[AirzoneUpdateCoordinator]):
+    """Define an Airzone Cloud entity."""
+
+    def get_airzone_value(self, key: str) -> Any:
+        """Return Airzone Cloud entity value by key."""
+        raise NotImplementedError()
```

Comment:
You could make this class an ABC and mark this method as abstract instead of raising if it will be implemented by all derived classes?